README.md

```diff
@@ -18,15 +18,27 @@ Engage with the community and get technical support on [the ODK-X forum](https:/
 
 ## Build 
 
+to build a docker image directly from the remote repository:
+
 `docker build --pull -t odk/sync-web-ui https://github.com/opendatakit/sync-endpoint-web-ui.git`
 
+alternatively, if you've cloned the repository to your local computer, nagivate to the directory containing the DockerFile, and use
+
+`docker build -t odk/sync-web-ui .`
+
 ## Run
 
-Use [RUN.md](RUN.md).
+The Web UI must run in conjunction with ODK sync-endpoint. 
+The fastest way to get sync-endpoint set up is through following the build and run instructions of
+[sync-endpoint-default-setup](https://github.com/opendatakit/sync-endpoint-default-setup).
+
+Once everything is set and ready, in the `sync-endpoint-default-setup` directory, run command
+`docker stack deploy -c docker-compose.yml syncldap` to deploy all services
```

Comment:
You should add some comments about the networking issues HTTP for local (likely) as HTTPS needs a certificate which you can't get for localhost.